tensorflow/core/api_def/python_api/api_def_CudnnRNNCanonicalToParamsV2.pbtxt

```diff
@@ -0,0 +1,4 @@
+op {
```

Comment:
tensorflow/core/api_def/api_test.cc:216
Value of: api_defs_map_.find(op.name()) != api_defs_map_.end()
  Actual: false
Expected: true
CudnnRNNCanonicalToParamsV2 op does not have api_def_*.pbtxt file. Please add api_def_CudnnRNNCanonicalToParamsV2.pbtxt file under tensorflow/core/api_def/base_api/ directory.